src/components/Comment/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { View, TouchableOpacity } from 'react-native';
+
+import Feather from '@expo/vector-icons/FontAwesome5';
+
+import {
+  Container,
+  ProfilePhoto,
+  ContentContainer,
+  Name,
+  Caption,
+  IconsContainer,
+  TimeToComment
+} from './styles';
+interface CommentProps {
+  name: string;
+  profile_image: string;
+  hours: string;
+  comment: string;
+  likes: string;
+}
+
+
+export function Comment({profile_image,name,comment,hours,likes}:CommentProps) {
+  return (
+      <Container>
+        <View style={{ width: '10%' }}>
+          <ProfilePhoto source={{ uri: `${profile_image}`  }} />
+        </View>
+        <ContentContainer>
+           <Name>{name}</Name>
+           <Caption>{comment}</Caption>
+        </ContentContainer>
+        <IconsContainer>
+          <TimeToComment>{hours}</TimeToComment>
+          <TouchableOpacity>
+              <Feather name="heart" color={'#E82656'} size={11}>
```

Comment:
increase the like icon a little, so it is more accessible for small cell phones, with that also increase the size of the other elements..